LocalWiki.php

```diff
@@ -421,3 +421,5 @@ function onBeforePageDisplay( OutputPage $out ) {
 
 		break;
 }
+
+$wgCargoDBname = $wgDBname . 'cargo';
```

Comment:
I don't have any issues with it being here, but there shouldn't be any issues with it being in LocalSettings instead. I'll leave it up to you anyway.